ffi/derive/src/impl_visitor.rs

```diff
@@ -287,10 +303,6 @@ impl<'ast> Visit<'ast> for FnVisitor<'ast> {
     fn visit_impl_item_method(&mut self, node: &'ast syn::ImplItemMethod) {
         self.doc = find_doc_attr(&node.attrs).cloned();
 
-        if !matches!(node.vis, syn::Visibility::Public(_)) {
```

Comment:
can we keep the check for associated methods? or do you think it's ok to allow export of private methods?

Reply:
You're right, we should keep this check.